packages/entity-browser/src/util/detailView/showBackButton.js

```diff
@@ -0,0 +1,8 @@
+const showBackButton = (initialKey, modelPaths) => {
+  if (modelPaths.length === 0) {
+    return (initialKey === '' || isNaN(initialKey))
```

Comment:
this requires the intialKey to be set in input? instead i would write return !initialKey || ... to make it optional